src/transaction/log_recovery_redo_parallel.hpp

```diff
@@ -38,6 +38,10 @@
 namespace cublog
 {
 #if defined(SERVER_MODE)
+  static constexpr size_t PARALLEL_RECOVERY_REDO_TUNING_REUSABLE_JOBS_STACK_SIZE = 100 * ONE_K;
```

Comment:
Keep the name to the shortest meaningful version.
```suggestion
  static constexpr size_t PARALLEL_REDO_JOBS_STACK_SIZE = 100 * ONE_K;
```